src/mrack/transformers/beaker.py

```diff
@@ -15,6 +15,7 @@
 """Beaker transformer module."""
 import re
 
+from mrack.context import global_context
```

Comment:
Global context should not be needed here to access provisioning config. Transformer should have it in `self.config` (provider) or `self._config` (global). I'd even say that using global context is an anti-pattern.

Reply:
removed in future commit